linalg/blockoperator.cpp

```diff
@@ -288,7 +288,7 @@ void BlockLowerTriangularPreconditioner::MultTranspose (const Vector & x,
       tmp2.SetSize(offsets[iRow+1] - offsets[iRow]);
       tmp2 = 0.0;
       tmp2 += xblock.GetBlock(iRow);
-      for (int jCol=iRow+1; jCol < iRow; ++jCol)
+      for (int jCol=iRow+1; jCol < nBlocks; ++jCol)
       {
          if (op(iRow,jCol))
```

Comment:
I know this is probably a square operator, but shouldn't these checks be transposed, e.g. `MFEM_ASSERT(x.Size() == height`?